core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2282,6 +2293,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_WRITE_CACHE_TTL =
+      new Builder(Name.USER_FILE_WRITE_CACHE_TTL)
```

Comment:
Does this apply to non-cache writes using THROUGH or CACHE_THROUGH write types? If so we should remove the "cache" part

Reply:
copy that